web/Controllers/AdminController.cs

```diff
@@ -247,17 +247,24 @@ public async Task<ActionResult> ImportTenants(IFormFile csvFile)
             return RedirectToAction(nameof(TenantList));
         }
         
-        var furtherFlashResponse = await _csvImportService.ImportTenants(csvFile);
-        for (int i = 0; i < furtherFlashResponse.FlashMessages.Count(); i++)
+        flashResponse = await _csvImportService.ImportTenants(csvFile);
+        for (int i = 0; i < flashResponse.FlashMessages.Count(); i++)
         {
-            AddFlashMessage(furtherFlashResponse.FlashTypes[i],  furtherFlashResponse.FlashMessages[i]);
+            AddFlashMessage(flashResponse.FlashTypes[i],  flashResponse.FlashMessages[i]);
         }
         
-        if (furtherFlashResponse.FlashMessages.Count() == 0)
+        if (flashResponse.FlashMessages.Count() == 0)
         {
             _logger.LogInformation("Successfuly imported all tenant data.");
             AddFlashMessage("success", "Successfully imported all tenant data.");
         }
+        
+        flashResponse =  _csvImportService.AddLatLonToTenantDb();
+        for (int i = 0; i < flashResponse.FlashMessages.Count(); i++)
+        {
+            AddFlashMessage(flashResponse.FlashTypes[i],  flashResponse.FlashMessages[i]);
+        }

```

Comment:
Could this be done as part of the main `ImportTenants` method?

Reply:
It could be merged into the final step (i.e. when the import is known to be succeeding), but I though it was clearer to separate it out.

Reply:
I don't mind either way tbh, it just felt like it was part of the overall import operation seeing as it updates the tenant table

Reply:
We still have to run over the table twice, since we're first adding the (missing) postcode to LatLon conversions to the postcode table and then using that table to add the LatLon data to the tenant table - I originally tried to do it all at once, but it caused database errors (it can't write to the Postcode table in the middle of writing to the Tenant table, or something to that effect).

Reply:
So there is a way to make this all work nicely, but I will leave that for a later ticket